ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/hooks/getTableUIState.ts

```diff
@@ -0,0 +1,80 @@
+import { SearchFilter } from 'types/search';
+
+type IdleState = {
```

Comment:
IMO I think it would be safe to export all of these types, I think at some point we are going to want to reference them directly.